plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlDistributedQueries.java

```diff
@@ -150,5 +146,16 @@ protected boolean isColumnNameRejected(Exception exception, String columnName, b
         return Optional.of(dataMappingTestSetup);
     }
 
+    @Test
+    @Override
+    public void testDelete()
+    {
+        // Mysql connector currently does not support row-by-row delete
+        String tableName = "test_delete_" + randomTableSuffix();
+        assertUpdate("CREATE TABLE " + tableName + " AS SELECT * FROM orders WITH NO DATA", 0);
+        assertQueryFails("DELETE FROM " + tableName + " WHERE random(orderkey) =  0", "Not supported delete");
+        assertUpdate("DROP TABLE " + tableName);
+    }
```

Comment:
i'd expect varchar-based delete to be special, since MySQL connector doesn't enforce varchar predicates (case sensitiviness). Why is it not special?